.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    open-pull-requests-limit: 10
+    schedule:
+      interval: daily
```

```suggestion
      interval: daily

```
[nit] may not be required

---

true, that's the default. Leaving it there for reference in case we want to change it